react/features/prejoin/middleware.js

```diff
@@ -29,8 +31,26 @@ MiddlewareRegistry.register(store => next => async action => {
 
         break;
     }
+
+    case SET_AUDIO_MUTED: {
```

Comment:
This change could go in its own commit, since it's not really related to exposing the Prejoin screen as an app.

Reply:
moved it into a separate commit, but kept it in the same pull request